.github/workflows/docker-release.yml

```diff
@@ -12,8 +12,20 @@ on:
       - 'yarn.lock'
 
 jobs:
+  check-env:
+    runs-on: ubuntu-latest
+    outputs:
+      check-docker: ${{ steps.check-docker.outputs.defined }}
+    steps:
+      - id: check-docker
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        if: ${{ env.DOCKER_USERNAME != '' }}
+        run: echo "::set-output name=defined::true"
   release:
     runs-on: ubuntu-latest
+    needs: check-env
+    if: needs.check-env.outputs.defined == 'true'
```

Comment:
if: needs.check-env.outputs.check-docker == 'true'

Reply:
should work like https://github.com/AgFlore/RSSHub/runs/5066207981 @TonyRL @liyishuai